docs/api/ref/DesktopAgent.md

```diff
@@ -411,12 +413,12 @@ await fdc3.raiseIntent("StartChat", context, appIntent.apps[0]);
 raiseIntentForContext(context: Context, app?: TargetApp): Promise<IntentResolution>;
 ```
 
-Finds and raises an intent against a target app based purely on context data.
+Finds and raises an intent against apps registered with the desktop agent based purely on the type of the context data.
 
-The desktop agent will resolve the correct app to target based on the provided context.
+The desktop agent will first resolve to a specific intent based on the provided context, displaying a resolver UI if more than one intent is available for specified context. It will then resolve to a specific app to handle the selected intent and specified context. 
+Alternatively, the specific app to target can also be provided, in which case the resolver should only offer intents supported by the specified application. 
```

Comment:
Again here, a resolver UI is not the only option.

Reply:
Can be reworded without mentioning UI.
```suggestion
Alternatively, the specific app to target can also be provided, in which case any method of resolution SHOULD only consider intents supported by the specified application. 
```